spec/system/admin/bulk_url_upload_spec.rb

```diff
@@ -66,6 +68,23 @@
         do_bulk_upload
         expect(reindexed_domains).to eq(Set[searchgov_domain])
       end
+
+      context 'when reindexing existing URLs' do
+        let(:reindex) { true }
+        let!(:existing_url) do
+          SearchgovUrl.create!(
+            url: 'https://agency.gov/',
+            last_crawled_at: 1.week.ago,
+            last_crawl_status: 'OK'
+          )
+        end
+
+        it 'reindexes the URLs' do
```

Comment:
Can an assertion be added here to check that reindexing is triggered for the URL, or is that not possible with the `except` in L19?